starwars/src/App.css

```diff
@@ -6,3 +6,32 @@
   text-shadow: 1px 1px 5px #fff;
 }
 /* write your parent styles in here for your App.js */
+
+
+.starwars-card{
+  border: 1px solid;
+  display: inline-flex;
+  flex-direction: column;
+  width: 550px;
+  margin: 0.5%;
+  border-radius: 10px;
+  background-color: rgba(114, 133, 145, 0.568);
+}
+.starwars-card .name{
+  font-weight: bold;
+  text-decoration: underline;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: tan;
+  height: 40px;
+  font-size: 25px;
+  border: 1px solid;
+}
+.starwars-card .info{
+  display: flex;
+}
+.starwars-card p{
```

Comment:
Joe, please be careful and make to match your CSS references to the appropriate className. After popping your work in CodeSandbox, I noticed the discrepancy as to why your CSS was not working. Inside of line 38 of your App.js file you name your container `div className='starwarscard'` . Here in your App.css you are referencing it as `.starwars-card` thus disabling all styling efforts. Correcting this error ensures your styling appears as originally intended.